x-pack/plugin/deprecation/qa/early-deprecation-rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/EarlyDeprecationIndexingIT.java

```diff
@@ -70,6 +70,7 @@ public void testEarlyDeprecationIsIndexedAfterTemplateIsLoaded() throws Exceptio
                 )
             );
         }, 30, TimeUnit.SECONDS);
+        fail();
```

Comment:
Looks like a left over from testing?

Reply:
ah yes :) at least it would be failing reliably lol :D